user_guide/src/examples/time_series/time_zones.py

```diff
@@ -0,0 +1,23 @@
+import polars as pl
+
```

Comment:
Can we split all these snippets to single files? This will ensure the line offsets are not getting messed up when we change/remove/add snippets.

Reply:
yup, good call!

I've done that, these are all now in their own `time_zones` folder

Waiting for a release to include https://github.com/pola-rs/polars/pull/6649, then I'll update to use that instead of `tz_localize`